grave-pile.lic

```diff
@@ -0,0 +1,36 @@
+custom_require.call(%w[events common-travel common])
+
+
+junk_list = get_settings.grave_junk.map { |x| /\b#{x}/i }
+
+DRCT.walk_to(10_291)
+loop do
+  break if checkbleeding
+  break if checkhealth < 75
+
+  until checkright || checkleft
+    fput('stand') unless checkstanding
```

Comment:
There's a `fix_standing` helper (https://github.com/rpherbig/dr-scripts/blob/master/common.lic#L223) you might want to use here